diff --git a/node_modules/@ecency/render-helper/lib/consts/allowed-attributes.const.js b/node_modules/@ecency/render-helper/lib/consts/allowed-attributes.const.js
index e2a8036..aef2cb2 100644
--- a/node_modules/@ecency/render-helper/lib/consts/allowed-attributes.const.js
+++ b/node_modules/@ecency/render-helper/lib/consts/allowed-attributes.const.js
@@ -16,6 +16,7 @@ exports.ALLOWED_ATTRIBUTES = {
         'data-youtube',
         'data-start-time',
         'data-video-href',
+        'data-is-inline',
         'data-proposal',
         'class',
         'title',
diff --git a/node_modules/@ecency/render-helper/lib/methods/a.method.js b/node_modules/@ecency/render-helper/lib/methods/a.method.js
index 2b54620..1637489 100644
--- a/node_modules/@ecency/render-helper/lib/methods/a.method.js
+++ b/node_modules/@ecency/render-helper/lib/methods/a.method.js
@@ -54,14 +54,19 @@ function a(el, forApp, webp) {
         var tag = postMatch[2];
         var author = postMatch[3].replace('@', '');
         var permlink = postMatch[4];
+        var isInline = true;
         if (el.textContent === href) {
             el.textContent = "@".concat(author, "/").concat(permlink);
+            isInline = false;
         }
         if (forApp) {
-            el.removeAttribute('href');
+            el.removeAttribute('href')
+
+            el.setAttribute('data-href', href)
             el.setAttribute('data-tag', tag);
             el.setAttribute('data-author', author);
             el.setAttribute('data-permlink', permlink);
+            el.setAttribute('data-is-inline', '' + isInline);
         }
         else {
             var h = "/".concat(tag, "/@").concat(author, "/").concat(permlink);
@@ -124,14 +129,20 @@ function a(el, forApp, webp) {
             el.setAttribute('class', 'markdown-post-link');
             var author = tpostMatch[2].replace('@', '');
             var permlink = tpostMatch[3];
+            var isInline = true;
             if (el.textContent === href) {
                 el.textContent = "@".concat(author, "/").concat(permlink);
+                isInline = false;
             }
             if (forApp) {
-                el.removeAttribute('href');
+                el.removeAttribute('href')
+                
+                el.setAttribute('data-href', href)
                 el.setAttribute('data-tag', tag);
                 el.setAttribute('data-author', author);
                 el.setAttribute('data-permlink', permlink);
+                el.setAttribute('data-is-inline', '' + isInline);
+           
             }
             else {
                 var h = "/".concat(tag, "/@").concat(author, "/").concat(permlink);
@@ -185,14 +196,21 @@ function a(el, forApp, webp) {
             var tag = 'post';
             var author = cpostMatch[1].replace('@', '');
             var permlink = cpostMatch[2];
+            var isInline = true;
+
             if (el.textContent === href) {
                 el.textContent = "@".concat(author, "/").concat(permlink);
+                isInline = false;
             }
             if (forApp) {
-                el.removeAttribute('href');
+                el.removeAttribute('href')
+
+                el.setAttribute('data-href', href)
                 el.setAttribute('data-tag', tag);
                 el.setAttribute('data-author', author);
                 el.setAttribute('data-permlink', permlink);
+                el.setAttribute('data-is-inline', '' + isInline);
+       
             }
             else {
                 var h = "/".concat(tag, "/@").concat(author, "/").concat(permlink);
@@ -273,14 +291,19 @@ function a(el, forApp, webp) {
         var tag = 'ccc';
         var author = cccMatch[2].replace('@', '');
         var permlink = cccMatch[3];
+        var isInline = true;
         if (el.textContent === href) {
             el.textContent = "@".concat(author, "/").concat(permlink);
+            isInline = false;
         }
         if (forApp) {
-            el.removeAttribute('href');
+            el.removeAttribute('href')
+
+            el.setAttribute('data-href', href)
             el.setAttribute('data-tag', tag);
             el.setAttribute('data-author', author);
             el.setAttribute('data-permlink', permlink);
+            el.setAttribute('data-is-inline', '' + isInline);
         }
         else {
             var h = "/".concat(tag, "/@").concat(author, "/").concat(permlink);
diff --git a/node_modules/@ecency/render-helper/src/consts/allowed-attributes.const.ts b/node_modules/@ecency/render-helper/src/consts/allowed-attributes.const.ts
index 2056f98..87b2d73 100644
--- a/node_modules/@ecency/render-helper/src/consts/allowed-attributes.const.ts
+++ b/node_modules/@ecency/render-helper/src/consts/allowed-attributes.const.ts
@@ -15,6 +15,7 @@ export const ALLOWED_ATTRIBUTES: XSSWhiteList = {
     'data-youtube',
     'data-start-time',
     'data-video-href',
+    'data-is-inline',
     'data-proposal',
     'class',
     'title',
diff --git a/node_modules/@ecency/render-helper/src/methods/a.method.ts b/node_modules/@ecency/render-helper/src/methods/a.method.ts
index 70ec473..4e44bd8 100644
--- a/node_modules/@ecency/render-helper/src/methods/a.method.ts
+++ b/node_modules/@ecency/render-helper/src/methods/a.method.ts
@@ -25,7 +25,8 @@ import {
   DOMParser,
   LOOM_REGEX,
   SECTION_REGEX,
-  SECTION_LIST
+  SECTION_LIST,
+  TRUVVL_REGEX
 } from '../consts'
 import { getSerializedInnerHTML } from './get-inner-html.method'
 import { proxifyImageSrc } from '../proxify-image-src'
@@ -101,15 +102,20 @@ export function a(el: HTMLElement, forApp: boolean, webp: boolean): void {
     const tag = postMatch[2]
     const author = postMatch[3].replace('@', '')
     const permlink = postMatch[4]
+    let isInline = true;
     if (el.textContent === href) {
       el.textContent = `@${author}/${permlink}`
+      isInline = false;
     }
     if (forApp) {
       el.removeAttribute('href')
 
+      el.setAttribute('data-href', href)
       el.setAttribute('data-tag', tag)
       el.setAttribute('data-author', author)
       el.setAttribute('data-permlink', permlink)
+      el.setAttribute('data-inline', 'testing')
+     
     } else {
       const h = `/${tag}/@${author}/${permlink}`
       el.setAttribute('href', h)
@@ -175,14 +181,20 @@ export function a(el: HTMLElement, forApp: boolean, webp: boolean): void {
       el.setAttribute('class', 'markdown-post-link')
       const author = tpostMatch[2].replace('@', '')
       const permlink = tpostMatch[3]
+      let isInline = true;
       if (el.textContent === href) {
         el.textContent = `@${author}/${permlink}`
+        isInline = false;
       }
       if (forApp) {
         el.removeAttribute('href')
+
+        el.setAttribute('data-href', href)
         el.setAttribute('data-tag', tag)
         el.setAttribute('data-author', author)
         el.setAttribute('data-permlink', permlink)
+        el.setAttribute('data-inline', 'testing')
+
       } else {
         const h = `/${tag}/@${author}/${permlink}`
         el.setAttribute('href', h)
@@ -241,14 +253,20 @@ export function a(el: HTMLElement, forApp: boolean, webp: boolean): void {
 
       const author = cpostMatch[1].replace('@', '')
       const permlink = cpostMatch[2]
+      let isInline = true;
       if (el.textContent === href) {
         el.textContent = `@${author}/${permlink}`
+        isInline = false;
       }
       if (forApp) {
         el.removeAttribute('href')
+
+        el.setAttribute('data-href', href)
         el.setAttribute('data-tag', tag)
         el.setAttribute('data-author', author)
         el.setAttribute('data-permlink', permlink)
+        el.setAttribute('data-inline', 'testing')
+   
       } else {
         const h = `/${tag}/@${author}/${permlink}`
         el.setAttribute('href', h)
@@ -337,15 +355,20 @@ export function a(el: HTMLElement, forApp: boolean, webp: boolean): void {
     const tag = 'ccc'
     const author = cccMatch[2].replace('@', '')
     const permlink = cccMatch[3]
+    let isInline = true;
     if (el.textContent === href) {
       el.textContent = `@${author}/${permlink}`
+      isInline = false;
     }
     if (forApp) {
       el.removeAttribute('href')
 
+      el.setAttribute('data-href', href)
       el.setAttribute('data-tag', tag)
       el.setAttribute('data-author', author)
       el.setAttribute('data-permlink', permlink)
+      el.setAttribute('data-is-inline', '' + isInline)
+
     } else {
       const h = `/${tag}/@${author}/${permlink}`
       el.setAttribute('href', h)
