diff --git a/node_modules/@esteemapp/react-native-autocomplete-input/index.js b/node_modules/@esteemapp/react-native-autocomplete-input/index.js
index bba35c7..4d134a6 100644
--- a/node_modules/@esteemapp/react-native-autocomplete-input/index.js
+++ b/node_modules/@esteemapp/react-native-autocomplete-input/index.js
@@ -7,9 +7,10 @@ import {
   Text,
   TextInput,
   View,
-  ViewPropTypes as RNViewPropTypes
 } from 'react-native';
 
+import { ViewPropTypes as RNViewPropTypes } from 'deprecated-react-native-prop-types';
+
 // Keep this line for downwards compatibility with RN.
 // eslint-disable-next-line react/forbid-foreign-prop-types
 const ViewPropTypes = RNViewPropTypes || View.propTypes;
