diff --git a/node_modules/@react-native-community/async-storage/android/build.gradle b/node_modules/@react-native-community/async-storage/android/build.gradle
index a0f2770..22072f9 100644
--- a/node_modules/@react-native-community/async-storage/android/build.gradle
+++ b/node_modules/@react-native-community/async-storage/android/build.gradle
@@ -2,7 +2,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral() //patched
     }
 
     dependencies {
