diff --git a/node_modules/@react-native-community/toolbar-android/android/build.gradle b/node_modules/@react-native-community/toolbar-android/android/build.gradle
index e414765..3c432e8 100644
--- a/node_modules/@react-native-community/toolbar-android/android/build.gradle
+++ b/node_modules/@react-native-community/toolbar-android/android/build.gradle
@@ -2,7 +2,7 @@
 buildscript {
     repositories {
         google()
-        jcenter()
+        mavenCentral()
     }
 
     dependencies {
