diff --git a/node_modules/react-native-image-viewing/dist/ImageViewing.d.ts b/node_modules/react-native-image-viewing/dist/ImageViewing.d.ts
index b279504..14a6cab 100644
--- a/node_modules/react-native-image-viewing/dist/ImageViewing.d.ts
+++ b/node_modules/react-native-image-viewing/dist/ImageViewing.d.ts
@@ -21,6 +21,7 @@ declare type Props = {
     backgroundColor?: string;
     swipeToCloseEnabled?: boolean;
     doubleTapToZoomEnabled?: boolean;
+    doubleTapScale?: number;
     delayLongPress?: number;
     HeaderComponent?: ComponentType<{
         imageIndex: number;
diff --git a/node_modules/react-native-image-viewing/dist/ImageViewing.js b/node_modules/react-native-image-viewing/dist/ImageViewing.js
index 3d7641a..ccdd063 100644
--- a/node_modules/react-native-image-viewing/dist/ImageViewing.js
+++ b/node_modules/react-native-image-viewing/dist/ImageViewing.js
@@ -18,7 +18,7 @@ const DEFAULT_BG_COLOR = "#000";
 const DEFAULT_DELAY_LONG_PRESS = 800;
 const SCREEN = Dimensions.get("screen");
 const SCREEN_WIDTH = SCREEN.width;
-function ImageViewing({ images, keyExtractor, imageIndex, visible, onRequestClose, onLongPress = () => { }, onImageIndexChange, animationType = DEFAULT_ANIMATION_TYPE, backgroundColor = DEFAULT_BG_COLOR, presentationStyle, swipeToCloseEnabled, doubleTapToZoomEnabled, delayLongPress = DEFAULT_DELAY_LONG_PRESS, HeaderComponent, FooterComponent, }) {
+function ImageViewing({ images, keyExtractor, imageIndex, visible, onRequestClose, onLongPress = () => { }, onImageIndexChange, animationType = DEFAULT_ANIMATION_TYPE, backgroundColor = DEFAULT_BG_COLOR, presentationStyle, swipeToCloseEnabled, doubleTapToZoomEnabled, doubleTapScale = 1.5, delayLongPress = DEFAULT_DELAY_LONG_PRESS, HeaderComponent, FooterComponent, }) {
     const imageList = useRef(null);
     const [opacity, onRequestCloseEnhanced] = useRequestClose(onRequestClose);
     const [currentImageIndex, onScroll] = useImageIndexChange(imageIndex, SCREEN);
@@ -49,7 +49,7 @@ function ImageViewing({ images, keyExtractor, imageIndex, visible, onRequestClos
         length: SCREEN_WIDTH,
         offset: SCREEN_WIDTH * index,
         index,
-    })} renderItem={({ item: imageSrc }) => (<ImageItem onZoom={onZoom} imageSrc={imageSrc} onRequestClose={onRequestCloseEnhanced} onLongPress={onLongPress} delayLongPress={delayLongPress} swipeToCloseEnabled={swipeToCloseEnabled} doubleTapToZoomEnabled={doubleTapToZoomEnabled}/>)} onMomentumScrollEnd={onScroll} 
+    })} renderItem={({ item: imageSrc }) => (<ImageItem onZoom={onZoom} imageSrc={imageSrc} onRequestClose={onRequestCloseEnhanced} onLongPress={onLongPress} delayLongPress={delayLongPress} swipeToCloseEnabled={swipeToCloseEnabled} doubleTapToZoomEnabled={doubleTapToZoomEnabled} doubleTapScale={doubleTapScale}/>)} onMomentumScrollEnd={onScroll} 
     //@ts-ignore
     keyExtractor={(imageSrc, index) => keyExtractor
         ? keyExtractor(imageSrc, index)
diff --git a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.d.ts b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.d.ts
index 0b1aa96..3d6354b 100644
--- a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.d.ts
+++ b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.d.ts
@@ -15,6 +15,7 @@ declare type Props = {
     delayLongPress: number;
     swipeToCloseEnabled?: boolean;
     doubleTapToZoomEnabled?: boolean;
+    doubleTapScale?: number;
 };
-declare const _default: React.MemoExoticComponent<({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled, doubleTapToZoomEnabled, }: Props) => JSX.Element>;
+declare const _default: React.MemoExoticComponent<({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled, doubleTapToZoomEnabled, doubleTapScale, }: Props) => JSX.Element>;
 export default _default;
diff --git a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.js b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.js
index 5fcaa9f..a34b5a1 100644
--- a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.js
+++ b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.android.js
@@ -16,7 +16,7 @@ const SWIPE_CLOSE_VELOCITY = 1.75;
 const SCREEN = Dimensions.get("window");
 const SCREEN_WIDTH = SCREEN.width;
 const SCREEN_HEIGHT = SCREEN.height;
-const ImageItem = ({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled = true, doubleTapToZoomEnabled = true, }) => {
+const ImageItem = ({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled = true, doubleTapToZoomEnabled = true, doubleTapScale = 1.5, }) => {
     const imageContainer = useRef(null);
     const imageDimensions = useImageDimensions(imageSrc);
     const [translate, scale] = getImageTransform(imageDimensions, SCREEN);
@@ -40,6 +40,7 @@ const ImageItem = ({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPre
         initialTranslate: translate || { x: 0, y: 0 },
         onZoom: onZoomPerformed,
         doubleTapToZoomEnabled,
+        doubleTapScale,
         onLongPress: onLongPressHandler,
         delayLongPress,
     });
diff --git a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.d.ts b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.d.ts
index 1dd5e0e..2551caa 100644
--- a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.d.ts
+++ b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.d.ts
@@ -15,6 +15,7 @@ declare type Props = {
     delayLongPress: number;
     swipeToCloseEnabled?: boolean;
     doubleTapToZoomEnabled?: boolean;
+    doubleTapScale?: number;
 };
-declare const _default: React.MemoExoticComponent<({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled, doubleTapToZoomEnabled, }: Props) => JSX.Element>;
+declare const _default: React.MemoExoticComponent<({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled, doubleTapToZoomEnabled, doubleTapScale, }: Props) => JSX.Element>;
 export default _default;
diff --git a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.js b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.js
index 0708505..aaf4369 100644
--- a/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.js
+++ b/node_modules/react-native-image-viewing/dist/components/ImageItem/ImageItem.ios.js
@@ -16,12 +16,12 @@ const SWIPE_CLOSE_VELOCITY = 1.55;
 const SCREEN = Dimensions.get("screen");
 const SCREEN_WIDTH = SCREEN.width;
 const SCREEN_HEIGHT = SCREEN.height;
-const ImageItem = ({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled = true, doubleTapToZoomEnabled = true, }) => {
+const ImageItem = ({ imageSrc, onZoom, onRequestClose, onLongPress, delayLongPress, swipeToCloseEnabled = true, doubleTapToZoomEnabled = true, doubleTapScale = 1.5, }) => {
     const scrollViewRef = useRef(null);
     const [loaded, setLoaded] = useState(false);
     const [scaled, setScaled] = useState(false);
     const imageDimensions = useImageDimensions(imageSrc);
-    const handleDoubleTap = useDoubleTapToZoom(scrollViewRef, scaled, SCREEN);
+    const handleDoubleTap = useDoubleTapToZoom(scrollViewRef, scaled, SCREEN, doubleTapScale);
     const [translate, scale] = getImageTransform(imageDimensions, SCREEN);
     const scrollValueY = new Animated.Value(0);
     const scaleValue = new Animated.Value(scale || 1);
diff --git a/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.d.ts b/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.d.ts
index 1df97a3..2ee6539 100644
--- a/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.d.ts
+++ b/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.d.ts
@@ -12,5 +12,5 @@ import { Dimensions } from "../@types";
  * This is iOS only.
  * Same functionality for Android implemented inside usePanResponder hook.
  */
-declare function useDoubleTapToZoom(scrollViewRef: React.RefObject<ScrollView>, scaled: boolean, screen: Dimensions): (event: NativeSyntheticEvent<NativeTouchEvent>) => void;
+declare function useDoubleTapToZoom(scrollViewRef: React.RefObject<ScrollView>, scaled: boolean, screen: Dimensions, doubleTapScale?: number): (event: NativeSyntheticEvent<NativeTouchEvent>) => void;
 export default useDoubleTapToZoom;
diff --git a/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.js b/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.js
index 112e0b2..2c5e4f4 100644
--- a/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.js
+++ b/node_modules/react-native-image-viewing/dist/hooks/useDoubleTapToZoom.js
@@ -7,12 +7,13 @@
  */
 import { useCallback } from "react";
 const DOUBLE_TAP_DELAY = 300;
+const DEFAULT_DOUBLE_TAP_SCALE = 1.5;
 let lastTapTS = null;
 /**
  * This is iOS only.
  * Same functionality for Android implemented inside usePanResponder hook.
  */
-function useDoubleTapToZoom(scrollViewRef, scaled, screen) {
+function useDoubleTapToZoom(scrollViewRef, scaled, screen, doubleTapScale = DEFAULT_DOUBLE_TAP_SCALE) {
     const handleDoubleTap = useCallback((event) => {
         var _a, _b, _c;
         const nowTS = new Date().getTime();
@@ -26,10 +27,10 @@ function useDoubleTapToZoom(scrollViewRef, scaled, screen) {
             // Zooming in
             // TODO: Add more precise calculation of targetX, targetY based on touch
             if (!scaled) {
-                targetX = pageX / 2;
-                targetY = pageY / 2;
-                targetWidth = screen.width / 2;
-                targetHeight = screen.height / 2;
+                targetX = pageX / doubleTapScale;
+                targetY = pageY / doubleTapScale;
+                targetWidth = screen.width / doubleTapScale;
+                targetHeight = screen.height / doubleTapScale;
             }
             // @ts-ignore
             (_c = scrollResponderRef) === null || _c === void 0 ? void 0 : _c.scrollResponderZoomTo({
diff --git a/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.d.ts b/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.d.ts
index a4e41f0..012cff1 100644
--- a/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.d.ts
+++ b/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.d.ts
@@ -12,8 +12,9 @@ declare type Props = {
     initialTranslate: Position;
     onZoom: (isZoomed: boolean) => void;
     doubleTapToZoomEnabled: boolean;
+    doubleTapScale?: number;
     onLongPress: () => void;
     delayLongPress: number;
 };
-declare const usePanResponder: ({ initialScale, initialTranslate, onZoom, doubleTapToZoomEnabled, onLongPress, delayLongPress, }: Props) => readonly [GestureResponderHandlers, Animated.Value, Animated.ValueXY];
+declare const usePanResponder: ({ initialScale, initialTranslate, onZoom, doubleTapToZoomEnabled, doubleTapScale, onLongPress, delayLongPress, }: Props) => readonly [GestureResponderHandlers, Animated.Value, Animated.ValueXY];
 export default usePanResponder;
diff --git a/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.js b/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.js
index 99dcbc7..437af41 100644
--- a/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.js
+++ b/node_modules/react-native-image-viewing/dist/hooks/usePanResponder.js
@@ -12,10 +12,10 @@ const SCREEN = Dimensions.get("window");
 const SCREEN_WIDTH = SCREEN.width;
 const SCREEN_HEIGHT = SCREEN.height;
 const MIN_DIMENSION = Math.min(SCREEN_WIDTH, SCREEN_HEIGHT);
-const SCALE_MAX = 2;
+const DEFAULT_DOUBLE_TAP_SCALE = 1.5;
 const DOUBLE_TAP_DELAY = 300;
 const OUT_BOUND_MULTIPLIER = 0.75;
-const usePanResponder = ({ initialScale, initialTranslate, onZoom, doubleTapToZoomEnabled, onLongPress, delayLongPress, }) => {
+const usePanResponder = ({ initialScale, initialTranslate, onZoom, doubleTapToZoomEnabled, doubleTapScale = DEFAULT_DOUBLE_TAP_SCALE, onLongPress, delayLongPress, }) => {
     let numberInitialTouches = 1;
     let initialTouches = [];
     let currentScale = initialScale;
@@ -89,7 +89,7 @@ const usePanResponder = ({ initialScale, initialTranslate, onZoom, doubleTapToZo
             if (doubleTapToZoomEnabled && isDoubleTapPerformed) {
                 const isScaled = currentTranslate.x !== initialTranslate.x; // currentScale !== initialScale;
                 const { pageX: touchX, pageY: touchY } = event.nativeEvent.touches[0];
-                const targetScale = SCALE_MAX;
+                const targetScale = doubleTapScale;
                 const nextScale = isScaled ? initialScale : targetScale;
                 const nextTranslate = isScaled
                     ? initialTranslate
@@ -218,8 +218,8 @@ const usePanResponder = ({ initialScale, initialTranslate, onZoom, doubleTapToZo
                 isDoubleTapPerformed = false;
             }
             if (tmpScale > 0) {
-                if (tmpScale < initialScale || tmpScale > SCALE_MAX) {
-                    tmpScale = tmpScale < initialScale ? initialScale : SCALE_MAX;
+                if (tmpScale < initialScale || tmpScale > doubleTapScale) {
+                    tmpScale = tmpScale < initialScale ? initialScale : doubleTapScale;
                     Animated.timing(scaleValue, {
                         toValue: tmpScale,
                         duration: 100,
