diff --git a/node_modules/react-native-camera/android/build.gradle b/node_modules/react-native-camera/android/build.gradle
index 4aed495..fc83d00 100644
--- a/node_modules/react-native-camera/android/build.gradle
+++ b/node_modules/react-native-camera/android/build.gradle
@@ -10,7 +10,6 @@ buildscript {
     repositories {
       google()
       mavenCentral()
-      jcenter()
     }
 
     dependencies {
@@ -56,7 +55,6 @@ android {
 
 repositories {
   google()
-  jcenter()
   mavenCentral()
   maven { url "https://jitpack.io" }
   maven {
