diff --git a/node_modules/react-native-navigation-bar-color/android/build.gradle b/node_modules/react-native-navigation-bar-color/android/build.gradle
index 24026a4..b1f1d05 100644
--- a/node_modules/react-native-navigation-bar-color/android/build.gradle
+++ b/node_modules/react-native-navigation-bar-color/android/build.gradle
@@ -1,6 +1,6 @@
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral() //patched
     }
 
     dependencies {
